Gemfile

```diff
@@ -14,7 +14,7 @@ gem 'rails', '~> 6.1.3'
 @proofer_gem ||= { github: '18F/identity-proofer-gem', ref: 'v2.8.0' }
 @saml_gem ||= { github: '18F/saml_idp', tag: 'v0.14.1-18f' }
 @telephony_gem ||= { github: '18f/identity-telephony', tag: 'v0.2.0' }
-@validations_gem ||= { github: '18F/identity-validations', tag: 'v0.4.0' }
+@validations_gem ||= { github: '18F/identity-validations', branch: 'oyk-update-specs' }
```

Comment:
Update this to tag `v0.5.0` once https://github.com/18F/identity-validations/pull/9 is merged.